tests/api_tests/test_onsite_verification_api.py

```diff
@@ -210,10 +243,11 @@ def test_onsite_verification(self, mock_pdr_resource_generator):
 
         # Verify the data dict arg from each mocked ResourceRecordSet(schema, data) created by the PDR generator,
         # triggered by POST /OnSite/Id/Verification requests
-        self.assertEqual(mock_pdr_resource_generator.call_count, 5)
-        payload_list = [payload_1, payload_2, payload_3, payload_4, payload_5]
-        for i in range(4):
+        self.assertEqual(mock_pdr_resource_generator.call_count, 6)
+        payload_list = [payload_1, payload_2, payload_3, payload_4, payload_5, payload_6]
+        for i in range(len(payload_list)):
             resource_dict = mock_pdr_resource_generator.call_args_list[i].args[1]
+            print(resource_dict)
```

Comment:
Can you remove this please.

Reply:
ahh, sure..completely forgot to take that out before pushing this